exporter/key_groups.go

```diff
@@ -173,7 +173,10 @@ for i=3,#ARGV do
   end
 end
 for i,key in ipairs(batch[2]) do
-  usage = redis.call("MEMORY", "USAGE", key)
+  local reply = redis.pcall("MEMORY", "USAGE", key)  
```

Comment:
This doesn't seem to work, see the tests:

```
=== RUN   TestKeyGroupMetrics/synchronous_with_unclassified_keys

time="2023-12-14T11:01:58Z" level=error msg="ERR Error running script (call to f_2111fe4ef69857c0b23529937bb692b02c475a9b): @user_script:20: user_script:20: attempt to index local 'reply' (a number value)"

    key_groups_test.go:159: Key group count metrics are not expected:

         Expected: map[string]int{"key_exp":5, "key_paul":1, "key_ringo":1, "unclassified":6}

        Actual: map[string]int{}

    key_groups_test.go:167: Key group memory usage metrics are not expected:

         Expected: map[string]bool{"key_exp":true, "key_paul":true, "key_ringo":true, "unclassified":true}

        Actual: map[string]bool{}

    key_groups_test.go:171: Unexpected number of distinct key groups, expected: 4, actual: 0

=== RUN   TestKeyGroupMetrics/synchronous_with_overflow_keys

time="2023-12-14T11:01:58Z" level=error msg="ERR Error running script (call to f_2111fe4ef69857c0b23529937bb692b02c475a9b): @user_script:20: user_script:20: attempt to index local 'reply' (a number value)"

    key_groups_test.go:159: Key group count metrics are not expected:

         Expected: map[string]int{"overflow":12, "test-stream":1}

        Actual: map[string]int{}

    key_groups_test.go:167: Key group memory usage metrics are not expected:

         Expected: map[string]bool{"overflow":true, "test-stream":true}

        Actual: map[string]bool{}

    key_groups_test.go:171: Unexpected number of distinct key groups, expected: 13, actual: 0

--- FAIL: TestKeyGroupMetrics (0.11s)
```

Reply:
oups, my bads i haven't tested on a version that was supporting MEMORY comand.
It should be better now by checking the returned type
1bc1b4584b122297aadb8ceb270376537f2fbe58

Reply:
Looks good now, thanks!